### DISCUSSION TITLE
Transfers stop after 30 minutes exactly on MacOS

### DISCUSSION THREAD

**Discussion by sophCrying** [transmission/transmission#3075] [Q&A]
Running transmission 3.00 on MacOS Monte Rey 12.1, and am having a strange problem where all transfers, upload and download, completely stop 30 minutes after launch. I can confirm its exactly 30 mins to the minute using little snitch network monitor. Restarting the program fixes it, so my temporary solution is an automator workflow that just quits and relaunches the app every 30 minutes, but that is not a great long term solution. Anyone else encountered this issue or know a potential fix?

**Comment by Coeur** [comment]
If your app is minimised, it might be App Nap and fixed with #874.
